bgpd/bgp_clist.c

```diff
@@ -748,6 +748,30 @@ bool community_list_exact_match(struct community *com,
 	return false;
 }
 
+bool community_list_any_match(struct community *com,
+			      struct community_list *list)
+{
+	struct community_entry *entry;
+	uint32_t val;
+	int i;
+
+	for (i = 0; i < com->size; i++) {
+		val = community_val_get(com, i);
+
+		for (entry = list->head; entry; entry = entry->next) {
+			if ((entry->style == COMMUNITY_LIST_STANDARD) &&
+			    (community_include(entry->u.com,
+					       COMMUNITY_INTERNET) ||
```

Comment:
Don't use COMMUNITY_INTERNET at all, it's going to be deprecated soon. More details https://github.com/opensourcerouting/frr/commit/81a57d81812115bdbab25168e86e509e2db607a1.

Reply:
done